src/wp-includes/rest-api/endpoints/class-wp-rest-url-details-controller.php

```diff
@@ -200,8 +217,9 @@ public function permissions_check() {
 	 *
 	 * @since 5.9.0
 	 *
-	 * @param string $url The website url whose HTML we want to access.
-	 * @return string|WP_Error The HTTP response from the remote URL, or an error.
+	 * @param string $url The website url whose HTML to access.
```

Comment:
Let's capitalize "URL" consistently here :)

Reply:
Fixed in commit. Thank you @SergeyBiryukov!